src/drop.js

```diff
@@ -14,10 +14,9 @@ var slice = require('./slice');
  * @category List
  * @see R.transduce
  * @sig Number -> [a] -> [a]
- * @sig Number -> String -> String
- * @param {Number} n The number of elements of `xs` to skip.
- * @param {Array} xs The collection to consider.
- * @return {Array}
+ * @param {Number} n
+ * @param {*} coll
```

Comment:
Maybe `list` instead of `coll` for these? Collection seems like a more general term to me that could include objects? This could be a useful distinction for ramda as there are operations that could make sense for one, the other, or both.